apps/prairielearn/src/models/question.sql

```diff
@@ -3,9 +3,18 @@ SELECT
   *
 FROM
   questions
-where
+WHERE
   id = $question_id;
 
+-- BLOCK select_question_by_uuid
+SELECT
+  *
+FROM
+  questions
+WHERE
+  course_id = $course_id
+  AND uuid = $uuid;
```

Comment:
I see that some of the moved versions of this function has `AND q.deleted_at IS NULL`. I assume it was deliberate to remove that here? I can imagine that we don't want this added restriction because the UUID really is unique, even over delete questions, and we might want to get a delete question here? I'm not quite sure when this would happen and it's not clear to me what the correct behavior is.

Reply:
That actually may no have been deliberate. Good catch, I'm inclined to add it back. I now see that there are other sites that can be changed to use this new model function, so I'll change those as well.

Reply:
Done in 4feb499.